testing/tests/DevExpress.ui.widgets/contextMenu.tests.js

```diff
@@ -89,6 +89,24 @@ QUnit.test("lazy rendering: not render overlay on init", function(assert) {
     assert.ok($itemsContainer.length, "overlay is defined");
 });
 
+QUnit.test("item click should not prevent document click handler", function(assert) {
+    var instance = new ContextMenu(this.$element, {
+            items: [{ text: "a" }]
+        }),
+        documentClickHandler = sinon.stub();
+
+    try {
```

Comment:
Why do you need try/finally here?

Reply:
It is necessary to detach document event listener after the test and prevent usage of this subscription in other tests